text2digits/text2digits.py

```diff
@@ -11,6 +11,60 @@
 ORDINAL_ENDINGS = [('ieth', 'y'), ('th', '')]
 
 
+class TextPreprocessor:
+    @staticmethod
+    def bigram(word1, word2):
+        '''
+        Returns the bigram similarity of 2 words
+        '''
+        word1 = word1.lower()
+        word2 = word2.lower()
+        word1_length = len(word1)
+        word2_length = len(word2)
+        pairs1 = []
+        pairs2 = []
+
+        for i in range(word1_length):
+            if i == word1_length - 1:
+                continue
+            pairs1.append(word1[i] + word1[i+1])
+
+        for i in range(word2_length):
+            if i == word2_length - 1:
+                continue
+            pairs2.append(word2[i] + word2[i+1])
+
+        similar = [word for word in pairs1 if word in pairs2]
+
+        return float(len(similar)) / float(max(len(pairs1), len(pairs2)))
+
+    @staticmethod
+    def get_similarity(item1, item2):
+        '''
+        Returns a number within the range (0,1) determining how similar
+        item1 is to item2. 0 indicates perfect dissimilarity while 1
+        indicates equality.
+        '''
+        return TextPreprocessor.bigram(item1, item2)
+    
+    @staticmethod
+    def get_match(word, collection):
+        '''
+        Returns the most syntactically similar word in the collection
+        to the specified word.
+        '''
+        match = None
+        max_similarity = 0
+        
+        for item in collection:
+            similarity = TextPreprocessor.get_similarity(word, item)
+            if similarity > max(max_similarity, 0.5):
```

Comment:
Why 0.5? Can you make this be a param sent to t2n that the user can control?

Reply:
Will do